src/services/game/changeButtonText.js

```diff
@@ -0,0 +1,14 @@
+export default function changeButtonText(modeRulesClassList) {
```

Comment:
remove defualt function, what excatly is `changeButton` and `changeButtonText`?

i think this component name is inprorriate